renderer/index.js

```diff
@@ -99,6 +99,14 @@ function init () {
       }
     } else if (e.which === 32) { /* spacebar pauses or plays the video */
       dispatch('playPause')
+    } else if ((e.ctrlKey || e.metaKey) && e.altKey && e.which === 39) {
+      dispatch('playbackJump', state.playing.currentTime + 10)
+    } else if ((e.ctrlKey || e.metaKey) && e.altKey && e.which === 37) {
+      dispatch('playbackJump', state.playing.currentTime - 10)
+    } else if ((e.ctrlKey || e.metaKey) && (e.which === 107 || e.which === 171)) { /* CMD || CTRL + DOM_VK_ADD || DOM_VK_PLUS  */
+      dispatch('setPlaybackRate', state.playing.playbackRate + 10)
```

Comment:
The 'steps' should be implemented in base2, standard players use 1x, 2x, 4x, 8x, 16x, so the function to modify the rate should be 
* increase:
`dispatch('setPlaybackRate', state.playing.playbackRate * 2)` 
* decrease:
`dispatch('setPlaybackRate', state.playing.playbackRate / 2)`